dnf/cli/commands/repoquery.py

```diff
@@ -363,6 +363,15 @@ def run(self):
         q = self.filter_repo_arch(self.opts, q)
         orquery = q
 
+        if self.opts.obsoletes:
+            out = set()
+            for pkg in q:
+                for i in pkg.obsoletes:
+                    out.add(str(i))
```

Comment:
I don't understand, why would you use ``--qf`` with ``--obsoletes``? ``--obsoletes`` doesn't print list of packages, does it?